cloudant-client/src/main/java/com/cloudant/client/api/query/Selector.java

```diff
@@ -0,0 +1,49 @@
+/*
```

Comment:
Did you remove `OperationOrExpression.java`? I couldn't see it in the diff but not sure if it's just a github quirk.

Reply:
Good spot, I thought I'd done a rename, but it looks like I somehow kept the original. Fixed in 3c4aa85cdc3ec79f01851f02027f3af454c2d09b